src/main/resources/synthetic.xml

```diff
@@ -20,6 +20,8 @@
         <property name="portalAppConcreteUid" required="false" label="Concrete Portal Application Identifier"
                   description="Required when deploying a IBM-API type portlet. Concrete Unique identifier for this portal application as defined in the portlet.xml"
                   category="Portal"/>
+        <property name="disableDeregisterOnUninstall" kind="boolean" required="true" label="Disable deregistration on uninstall"
```

Comment:
When used in the deployable (wp.War), it is best to have this field as not required and of kind string.  Then you will also be able to use a placeholder and drive the functionality from dictionaries if required.